src/app/components/molecules/footer/footer.tsx

```diff
@@ -1,41 +1,70 @@
-/* eslint-disable jsx-a11y/anchor-is-valid */
-export async function NHSNotifyFooter() {
+import Link from 'next/link';
```

Comment:
Why are we using Next links?

Reply:
We are using NextJS so it is recommended to use `Link` tag rather than `a` tag because it provides built-in functions that helps improve performance and SEO.